components/clarinet-cli/src/devnet/package.rs

```diff
@@ -0,0 +1,128 @@
+use std::fs::{read_to_string, File};
+use std::io::{self, Write};
+use std::process;
+
+use clarinet_deployments::types::DeploymentSpecificationFile;
+use clarinet_files::NetworkManifestFile;
+
+use toml::value::Value;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Project {
+    name: String,
+    description: Option<String>,
+    authors: Vec<String>,
+    telemetry: Option<bool>,
+    cache_dir: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ClarinetSpecificationFile {
+    project: Project,
+    contracts: Option<Value>,
+    repl: Option<Value>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ConfigurationPackage {
+    deployment_plan: DeploymentSpecificationFile,
+    devnet_config: NetworkManifestFile,
+    clarinet_config: ClarinetSpecificationFile,
+}
+
+fn get_devnet_config() -> NetworkManifestFile {
```

Comment:
The `NetworkManifest` struct has a `from_project_manifest_location` function that you may be able to reuse:
https://github.com/hirosystems/clarinet/blob/6b35ed75cdd45731b746b8d76c9fe3f085f308c2/components/clarinet-files/src/network_manifest.rs#L286